test/Microsoft.Azure.ServiceBus.KeyVault.Test/EncryptionTests.cs

```diff
@@ -8,7 +8,8 @@ namespace Microsoft.Azure.ServiceBus.KeyVault.Test
     using System.Threading.Tasks;
     using Microsoft.Azure.ServiceBus.KeyVault;
     using Xunit;
-
+    using Microsoft.Azure.ServiceBus.Test.Shared;
```

Comment:
I'll have to push back on this one. Sorting of namespace doesn't heart readability of the code. Rather than enforcing that to be done manually, it's a work a tool can do 😉